### ISSUE TITLE
TinyGAN: Distilling BigGAN for Conditional Image Generation

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#135]
# TL;DR
A study to distill BigGANs. The latent and class variable input and image output pairs are obtained beforehand and treated as a dataset to reduce memory usage during training, and a small student model is trained with three targets: the L1 distance per pixel, the difference between the hidden layers of D, and the usual adversarial losses. Although the performance is slightly degraded, they succeeded in reducing the number of parameters compared to BigGANs.
![image](https://user-images.githubusercontent.com/24524018/95273794-f0930300-087e-11eb-9d2c-a4b7f01e0031.png)

![image](https://user-images.githubusercontent.com/24524018/95273796-f25cc680-087e-11eb-9e1d-e842eff137d9.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.13829

# Submission Dates(yyyy/mm/dd)
2020/09/29

# Authors and institutions

Ting-Yun Chang, Chi-Jen Lu
- Institute of Information Science

# Methods

# Results

# Comments
